src/plugins/intel_gpu/src/graph/convolution.cpp

```diff
@@ -19,6 +19,46 @@ primitive_type_id convolution::type_id() {
     return &instance;
 }
 
+static format GetRecommendFormat(layout input_layout, data_types output_type) {
```

Comment:
Why did you use camel case here? I think the convention here is to use underscore..